### ISSUE TITLE
Homepage switching not working on select themes

### ISSUE THREAD

**Issue by tanjoymor** [Automattic/themes#6660]
### Quick summary

When switching to some of the "Hybrid" themes, we're not presenting the choice to retain the Homepage vs load the theme's Homepage and instead we are by default moving the existing Homepage to drafts and pulling in the theme's demo Homepage.

This happens with **Stratford**, **Balasana**, **Mayland**, **Dalston**, and **Leven**. All other themes here: https://wordpress.com/themes/filter/hybrid-themes present the choice of what to do with the Homepage. 

### Steps to reproduce

1. On a Simple or AT site switch to any of the five themes: **Stratford**, **Balasana**, **Mayland**, **Dalston**, and **Leven**
2. The switch will immediately happen with no prompts

### What you expected to happen

I expected to be presented with the same "Homepage option" that all other Hybrid themes offer: ![llFPhs.png](https://user-images.githubusercontent.com/42655541/193436675-40dd1f01-1ac4-490d-9001-772d1d254dc6.png)

### What actually happened

The theme's Homepage was created and the existing Homepage was moved to drafts – without any prompting or warning.

### Context

Came across this during some routine testing.

### Platform (Simple, Atomic, or both?)

Simple, Atomic

### Theme-specific issue?

Stratford, Balasana, Mayland, Dalston, and Leven

### Browser, operating system and other notes

Tested in Chrome and Firefox

### Reproducibility

Consistent

### Severity

Some (< 50%)

### Available workarounds?

Yes, easy to implement

### Workaround details

Customer's are able to reinstate their existing Homepage, but it causes confusion (as it did before the Homepage switcher option was introduced on all of these themes). It's not so much whether the workaround on this is easy or difficult, it's not intuitive and causes confusion and sometimes panic.

All users who try to use these themes are affected, but not all users are trying to use these themes.

**Comment by jamiepalatnik** [comment]
📌 SCRUBBING : RESULT
- Tested on Simple - Replicated

📌 FINDINGS/SCREENSHOTS/VIDEO
- I was able to replicate this on all of the listed themes, as well as on the theme Varia. 
- I tested a few other themes from the hybrid themes filter and saw the modal with an option to choose if I wanted to use the theme homepage or not.
- I would also expect the experience to be consistent for these themes.

📌 ACTIONS
- Marked as Triaged for Quality Squad review

**Comment by tanjoymor** [comment]
These 5 themes are missing the "Theme Feature" tag of "Auto Loading Homepage" 

Dalston example:
![mKbsWq.png](https://user-images.githubusercontent.com/42655541/194678636-287554dc-23ac-4b27-a0ce-f3f8ba6dfdd4.png)

Rockfield example:
![DihKo8.png](https://user-images.githubusercontent.com/42655541/194678644-28ae692d-4e4a-46b9-afaf-6c83a5fcb7a4.png)

Does it break something if that tag is added to these themes?

**Comment by fushar** [comment]
Looks like it should be just missing the "Auto Loading Homepage" feature as mentioned above. @Automattic/lego let's fix this as part of the year-end maintenance.

**Comment by arthur791004** [comment]
Note that we have to take care of whether we use "Auto Loading Homepage" somewhere and whether we want the same behavior for  these themes (Stratford, Balasana, Mayland, Dalston, and Leven)

**Comment by fushar** [comment]
Interesting, looks like the `auto-loading-homepage` term is being used to query our "recommended" themes. Let me dig further...

**Comment by fushar** [comment]
After some investigation, I believe it is safe to add  the `auto-loading-homepage` term to all the mentioned themes. We only use the term in another place: the onboarding the Design Picker. But, the themes also need to be FSE in order for them to show up in the Design Picker. As these are all Hybrid themes, they won't show up.

So, adding `auto-loading-homepage` is safe. I've added it to the themes. I've confirmed that the theme switch modal is shown. Closing this issue now!
